src/transport_ofi.c

```diff
@@ -563,36 +615,71 @@ static inline int atomic_limitations_check(void)
     }
 
     /* Binary OPS check */
-    for(i=0; i<3; i++) {//DT
-      for(j=0; j<3; j++) { //OPS
+    for(i=0; i<SIZEOF_DT_INT; i++) {//DT
+      for(j=0; j<SIZEOF_BOPS; j++) { //OPS
```

Comment:
I know this is obnoxious feedback, but these comments are not very helpful.  I think this is a helpful spot for comments, these are just too brief.  Could you expand to something like ```/* Loop over bitwise ops and supported datatypes */```?